src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/TrackingOptInPage.java

```diff
@@ -117,6 +129,21 @@ private void isTrackingRequestsNotSend(
       List<String> elementsList, NetworkTrafficInterceptor networkTrafficInterceptor
   ) {
     wait.forX(WAITING_TIME_FOR_ALL_REQUESTS);
+    isConsentCookieSetToRejected();
+    for (String anElementsList : elementsList) {
+      Assertion.assertFalse(isSuccessfulResponseByUrlPattern(networkTrafficInterceptor,
+                                                             anElementsList
+                            ),
+                            "Request to " + anElementsList + " services was found"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Request to " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)